go.mod

```diff
@@ -3,12 +3,18 @@ module sigs.k8s.io/kubebuilder/v3
 go 1.15
 
 require (
+	github.com/cloudflare/cfssl v1.5.0
 	github.com/gobuffalo/flect v0.2.2
+	github.com/markbates/pkger v0.17.1
 	github.com/onsi/ginkgo v1.12.0
 	github.com/onsi/gomega v1.9.0
 	github.com/spf13/afero v1.2.2
-	github.com/spf13/cobra v0.0.7
+	github.com/spf13/cobra v1.0.0
 	github.com/spf13/pflag v1.0.5
-	golang.org/x/tools v0.0.0-20200403190813-44a64ad78b9b
+	golang.org/x/tools v0.0.0-20190920225731-5eefd052ad72
+	k8s.io/api v0.20.2 // indirect
+	k8s.io/apimachinery v0.20.2
+	sigs.k8s.io/controller-tools v0.3.0
+	sigs.k8s.io/kustomize/kyaml v0.10.10
```

Comment:
I am not sure if we should add that as dep of kb. Could we make it specific for config-gen instead? WDYT

Reply:
Are you suggesting making config-gen a separate binary?

Reply:
It might be a good idea if the goal is to have the config-gen such as controller-gen from the controller-tools. However, if you decede to do not move in this way. IMO we need to remove the indirect deps here and add a comment to describe what deps have been added only because of the config-gen.

Reply:
What is the concern here?

I'd rather not make it a separate binary as it creates friction both for development and users consuming it.  I could comment those dep lines, but go has a tool which explains the reason for transitive deps and the comment could get stale.

Reply:
I've cleaned up the file a bit by manually deleting the indirects and the tidy seems to keep them.  How does it look?